wp-content/themes/missilethreat/assets/_scss/components/_post-block.scss

```diff
@@ -1 +1,141 @@
 @use 'abstracts' as *;
+
+.post-block {
+  margin: rem(40px) 0;
+  color: $color-text-dark-170;
+  display: grid;
+  grid-template-columns: 1fr;
+
+  &:first-of-type {
+    margin: rem(24px) 0 rem(40px) 0;
+  }
+
+  &__category {
+    @extend %font-ui-text-sm;
+    display: inline-block;
+    margin-bottom: rem(16px);
+    grid-row: 1;
+
+    a:not(:last-child) {
+      margin-right: rem(4px);
+    }
+  }
+
+  &__title {
+    @extend %font-ui-headline-lg;
+    margin-bottom: rem(8px);
+    color: $color-text-dark-100;
+    grid-row: 2;
+  }
+
+  .post-meta__date {
+    grid-row: 3;
+  }
+
+  .post-meta__authors {
+    grid-row: 4;
+    margin-bottom: rem(16px);
+  }
+
+  .wp-post-image {
+    grid-row: 6;
+    margin-bottom: rem(24px);
+  }
+
+  &__excerpt {
+    @extend %font-body-short-md-serif;
+    color: $color-text-dark-190;
+    grid-row: 7;
+  }
+
+  @include breakpoint('medium') {
+    grid-template-columns: 2fr 1fr;
+    column-gap: rem(40px);
+    position: relative;
+
+    &__category,
+    &__title,
+    .post-meta__date,
+    .post-meta__authors,
+    &__excerpt {
+      grid-column: 1 / span 1;
+    }
+
+    .wp-post-image {
+      position: absolute;
+      grid-column: 2 / span 1;
+      grid-row: 1;
+      max-width: rem(200px);
+    }
+  }
+}
+
+.home {
```

Comment:
I would move this to the home.scss file. This is structural kind of stuff that is only relevant on the homepage, so you don't want to be carrying it around with you on all the other pages unnecessarily.